src/styles/components/multiple-form.less

```diff
@@ -160,6 +160,20 @@ base/forms.less
 
   }
 
+  .service-deploy-modal {
```

Comment:
Can we generalise this style, so that we can use the same for the create job modal?